jinahub/encoders/text/AudioCLIPTextEncoder/README.md

```diff
@@ -2,80 +2,77 @@
 
 **AudioCLIPTextEncoder** is an encoder that encodes text using the [AudioCLIP](https://arxiv.org/abs/2106.13043) model.
 
-This encoder is meant to be used in conjunction with the [AudioCLIPImageEncoder](https://github.com/jina-ai/executors/tree/main/jinahub/encoders/image/AudioCLIPImageEncoder) and [AudioCLIPEncoder](https://github.com/jina-ai/executors/tree/main/jinahub/encoders/audio/AudioCLIPEncoder), as it can embedd text, images and audio to the same latent space.
+Before using it, please check the [prerequisites](#prerequisites).
 
-You can use either the `Full` (where all three heads were trained) or the `Partial` (where the text and image heads were frozen) version of the model.
+This encoder is meant to be used in conjunction with the [AudioCLIPImageEncoder](https://hub.jina.ai/executor/3atsazub)
+and [AudioCLIPEncoder](https://hub.jina.ai/executor/f4d22e1r) encoders, as they embed text, images and audio to the same
+latent space.
 
-The following arguments can be passed on initialization:
+You can either use the `Full` (where all three heads were trained) or the `Partial` (where the text and image heads were
+frozen) versions of the model.
 
-- `model_path`: path of the pre-trained AudioCLIP model.
-- `default_traversal_paths`: default traversal path (used if not specified in request's parameters)
-- `default_batch_size`: default batch size (used if not specified in request's parameters)
-- `device`: device that the model is on (should be "cpu", "cuda" or "cuda:X", where X is the index of the GPU on the machine)
+For more information, such as how to run an Executor on a GPU,
+check [this guide](https://docs.jina.ai/tutorials/gpu-executor/).
 
 ## Prerequisites
 
-First, you should download the model and the vocabulary, which will be saved into the `.cache` folder inside your current directory (will be created if it does not exist yet).
+> These are only needed if you use it with the `jinahub://AudioCLIPTextEncoder` syntax.
```

Comment:
Since the `download_model` param is set to `False` by default, then these steps will be applicable to all usage type - with/without docker. One has to download and configure the `model_path`. 
We should either re-write this or remove it all-together (in other READMEs too).

Reply:
I clarified it